test/test_external.py

```diff
@@ -146,25 +150,25 @@ def test_english_to_spanish(self):
 
     def test_sentiment_model(self):
         interface_info = gr.external.load_interface("models/distilbert-base-uncased-finetuned-sst-2-english", alias="sentiment_classifier")
-        io = gr.Interface(**interface_info)
+        io = gr.Interface(**interface_info, analytics_enabled=False)
```

Comment:
Why do you have to do this on top of the env variable?

Reply:
You don't need, I started doing this and then realized we should make it an env. I'll fix